src/stuf/stuf_bg/tests/test_client.py

```diff
@@ -37,98 +34,73 @@ def _stuf_bg_xmlschema_doc():
         return etree.parse(infile)
 
 
-class StufBGConfigTests(TestCase):
-    def setUp(self):
-        super().setUp()
-        self.service = StufServiceFactory.create(soap_service__soap_version="1.1")
-        self.config = StufBGConfig.get_solo()
-        self.config.service = self.service
-        self.config.save()
-        self.client = self.config.get_client()
-
-    @freeze_time("2020-12-11T10:53:19+01:00")
-    @patch(
-        "stuf.client.uuid.uuid4",
-        return_value=uuid.UUID("38151851-0fe9-4463-ba39-416042b8f406"),
+def _extract_soap_body(full_doc: str):
+    doc = etree.parse(BytesIO(full_doc))
```

Comment:
note that there's also `stuf.xml.fromstring` which uses a defused parser and might make it easier to deal with the BytesIO stuff? It takes str and bytes